kafka/adminapi_test.go

```diff
@@ -464,7 +467,10 @@ func testAdminAPIsListConsumerGroups(
 		t.Fatalf("Expected ConsumerGroupTypeFromString to work for Unknown type")
 	}
 
-	listres, err := a.ListConsumerGroups(
+	ctx, cancel := context.WithTimeout(context.Background(), expDuration)
```

Comment:
I think this code doesn't run. Are the tests running when you are dynamically linking librdkafka? This is a compilation error because := and not =